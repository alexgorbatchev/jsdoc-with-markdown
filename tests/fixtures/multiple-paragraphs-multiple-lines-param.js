/**
 * This is a description.
 * And this is more of it.
 *
 * @param {Boolean} isAdditive -  Set to true to allow the dropdown to add in custom values.
 * When `dropdownMode` is set to false, `isAdditive` is always true.
 *
 * This is a second paragraph
 * with two lines.
 */
