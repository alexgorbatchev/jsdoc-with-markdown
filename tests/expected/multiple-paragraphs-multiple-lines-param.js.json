[
  {
    "type": "paragraph",
    "value": "<p>This is a description. And this is more of it.</p>\n"
  },
  {
    "type": "param",
    "param": {
      "type": "Boolean",
      "name": "isAdditive",
      "description": "<p>Set to true to allow the dropdown to add in custom values. When <code>dropdownMode</code> is set to false, <code>isAdditive</code> is always true.</p>\n"
    }
  },
  {
    "type": "paragraph",
    "value": "<p>This is a second paragraph with two lines.</p>\n"
  }
]
